modules/polling/components/PollOverviewCard.tsx

```diff
@@ -55,6 +55,13 @@ export default function PollOverviewCard({
     setCategoryFilter({ ...categoryFilter, [category]: !(categoryFilter || {})[category] });
   }
 
+  const myComment = comments?.find(c => {
+    return c.comment.hotAddress.toLowerCase() === account?.toLowerCase();
```

Comment:
Will this work with delegates? Can't remember if we store the delegate contract address or the delegate owner hot address as the comment address.  cc: @rafinskipg

Reply:
My goerli test account is a delegate, and it was showing up as expected, but I'm not sure if the Goerli db and prod db have a different schema? @rafinskipg are there any other considerations with addresses in the db?

Reply:
hey guys, hot address and account should be the same.

And there's no different schema, as long as you store anything through the same deployment, you should get the same data back in both databases